src/applications/personalization/appointments/actions/index.js

```diff
@@ -153,22 +153,19 @@ export function fetchConfirmedFutureAppointments() {
         {},
       );
     } catch (error) {
+      const errors = error.errors ?? [error];
       dispatch({
         type: FETCH_CONFIRMED_FUTURE_APPOINTMENTS_FAILED,
-        errors: [
-          ...(vaAppointmentsResponse?.errors || []),
```

Comment:
*This was the problem.* If this `catch` block was reached, it was because one of the `*Response` objects was never set because the async API call used to set it failed.